docs/_docs/langevin.md

```diff
@@ -0,0 +1,49 @@
+
+# Deterministic and Stochastic Dynamics
+The possibility of conducting hybrid molecular dynamics - Monte Carlo (MDMC) schemes is possible by addition of the "Langevin move" in the `moves` section.
+
+## Langevin Dynamics
+
+`langevin`      | Description
+--------------- | --------------------------------------------
+`splitting`     | Method for numerically solving the Langevin equation
+`nstep`         | Number of random MC steps to be conducted before continuing Langevin dynamics
+`mdsteps`       | Number of iterations for each Langevin dynamics event
+`dt`            | Time step for integration (ps)
+`gamma`         | Friction coefficient (1/ps)
+
+This move will solve the Langevin equation for the particles in the system on the form
+$$
+d\begin{bmatrix} \mathbf{q} \\ \mathbf{p} \end{bmatrix} = \underbrace{\begin{bmatrix} M^{-1}\mathbf{q} \\ 0 \end{bmatrix} \mathrm{d}t}_{\text{A}} + \underbrace{\begin{bmatrix} 0 \\ -\nabla U(\mathbf{q}) \end{bmatrix}\mathrm{d}t}_{\text{B}} + \underbrace{\begin{bmatrix} 0 \\-\gamma \mathbf{p} + \sqrt{2k\text{k}_{\mathrm{b}}T\gamma} \sqrt{M} ~\mathrm{d}\mathbf{W} \end{bmatrix}}_{\text{O}}
+$$
+Where A, B, and O makes up the individual terms for solving the Langevin equation, which can individually be solved exactly to obtain a trajectory given by
+$$
+\begin{align}
+\varphi^{\mathrm{A}}(\mathbf{q}, \mathbf{p}) &= \left(\mathbf{q} + \Delta t \sqrt{M}~ \mathbf{p}, \mathbf{p}\right) \\
+\varphi^{\mathrm{B}}(\mathbf{q}, \mathbf{p}) &= \left(\mathbf{q}, \mathbf{p} - \Delta t \nabla U(\mathbf{q})\right) \\
+\varphi^{\mathrm{O}}(\mathbf{q}, \mathbf{p}) &= \left(\mathbf{q}, e^{-\gamma \Delta t}\mathbf{p} + \sqrt{\mathrm{k}_{\mathrm{B}}T (1 - e^{-2\gamma \Delta t})} \sqrt{M} \mathbf{R} \right)
+\end{align}
+$$
+The keyword `splitting` thus refers to the string constructed from the labels A, B and O. In the current implementation a selection of splitting schemes are available
+
+`splitting`     | Description
+--------------- | --------------------------------------------
+ABAO            | [Geometric Langevin Position-Verlet](http://doi.org/fh8jpp)
+BABO            | [Geometric Langevin Velocity-Verlet](http://doi.org/fh8jpp)
+ABOBA           | [Symmetric Langevin Position-Verlet](http://doi.org/ggrnfs)
+BAOAB           | [Symmetric Langevin Velocity-Verlet](http://doi.org/ggrnfs)
+OABAO           | [Bussi-Parrinello Langevin Position-Verlet](http://doi.org/cjpdjx)
+OBABO           | [Bussi-Parrinello Langevin Velocity-Verlet](http://doi.org/cjpdjx)
+
+We recommend the usage of the splitting scheme "BAOAB" (Symmetric Langevin Velocity-Verlet) due to it being less errorprone with increasing timestep [Leimkuhler & Matthews, pp. 279-281](http://doi.org/dx7v).
+
+For examples of the usage of the Langevin move and combined usage with Monte Carlo moves see the [molecular dynamics](https://github.com/mlund/faunus/blob/master/examples/moleculardynamics) directory.
+
+## Molecular dynamics
```

Comment:
Remove for now?

Reply:
Possible include til line 19 and then give a yaml example?

Reply:
MD completely removed from the manual for now. The markup was also fixed to compile to PDF.